src/AirshipBindings.iOS.common/build.gradle

```diff
@@ -33,3 +33,42 @@ task build {
 }
 
 build.dependsOn('carthageUpdate')
+
+task carthageCheckout {
+    dependsOn 'syncVersion'
+    doLast() {
+        exec {
+            workingDir "$rootDir"
+            commandLine "carthage", "checkout"
+        }
+    }
+}
+
+task checkoutXamboni {
+    doLast() {
+        // Xamboni
+        def tmpDir = System.getProperty("java.io.tmpdir")
+        def xamboniTempRepo = new File("${tmpDir}/xamboni")
+        project.delete(files(xamboniTempRepo))
+
+        xamboniTempRepo.deleteOnExit()
+
+        println("Cloning Xamboni repo")
+        exec {
+            commandLine "git", "clone", "git@github.com:urbanairship/xamboni.git", "${xamboniTempRepo}"
+        }
+
+        checkoutXamboni.ext.repoPath = xamboniTempRepo
+    }
+}
+
+task cleanXamboniRepo(type: Delete) {
```

Comment:
Does this need to be removed?

Reply:
This gives a way for devs to make sure they have the newest Xamboni, by running a clean build.